### ISSUE TITLE
[feature request] custom NBT support via json like WailaNBT

### ISSUE THREAD

**Issue by ghost** [asaskevich/SmartCursor#17]
The idea is instead of having mod support people can make their own scripts to support visualizing mod contents using NBT data. 

See here for ex https://mods.curse.com/mc-mods/minecraft/224417-waila-nbt

The script would be something like that mod does it

When you have the specific item in your inventory  "divinationSigil then you can see Current Essence as "Amount" NBT tag value and "Capacity" as "capactity" NBT tag value on any block called containerAltar. EZ 

```
{
    "AWWayofTime:divinationSigil": { 
        "containerAltar": {
            "Amount": "Current Essence",
            "capacity": "Capacity"
        }
    }
}

```

If you literarily copy /pasta WailaNBT json system then you can use all their scripts which support most of the major mods already. 

Not only that but anyone can make advanced stuff like % growth of crops or how much liquid is in a cauldron or light level of a block and so forth. And would work for ANYTHING and best of all you won't have to manually keep updating your mod to stay compatible with other mods. 

I don't know how hard that would be to implement but it would save you a lot of trouble in the long run making this mod way more versatile than WAILA since that WailaNBT only works for 1.7.10 version.

In short people can make their own scripts for whatever they need  to visualize in the tooltips.

**Comment by asaskevich** [comment]
Great idea, will look how it can be done at the current moment
